CONTRIBUTING.md

```diff
@@ -151,6 +152,11 @@ Validate authentication using the username `alice` with the password `alice`:
 $ curl -v -k -X POST -d "alice" http://localhost:3000/authn-ldap/test/cucumber/alice/authenticate
 ```
 
+#### Google Cloud Platform (GCP) Authentication
+
+To enable a host to log into Conjur using GCP identity token, run `start` with the `--authn-gcp` flag.
```

Comment:
drop down into the `dev` folder and run `start`

Reply:
@eranha not a much but would save time to say that they need to be in dev folder to run start